javascript/javascript2/week3/game.js

```diff
@@ -0,0 +1,103 @@
+let gameStart = false;
+let sCount = 0;
+let lCount = 0;
+let counter = 0;
+let isKeyPress = false;
+confetti = new ConfettiGenerator({ target: 's' });
+const h1 = document.getElementById('screen-msg');
+const duration = document.getElementById('duration');
+const button1 = document.getElementById('start');
+const winner = (sCount, lCount) => {
+    if (sCount > lCount) {
+        document.getElementById('msg-s').innerHTML = `Winner`;
+        confetti = new ConfettiGenerator({ target: 's' });
+        confetti.render();
+    }
+    else if (lCount > sCount) {
+        document.getElementById('msg-l').innerHTML = `Winner`;
+        confetti = new ConfettiGenerator({ target: 'l' });
+        confetti.render();
+    }
+    else {
+        document.getElementById('msg-s').innerHTML = `draw`;
+        document.getElementById('msg-l').innerHTML = `draw`;
+
+    }
+}
+
+
+button1.addEventListener('click', countDown => {
+    confetti.clear();
+    let duration1 = parseInt(duration.value);
+    counter = duration1 - 1;
+    if (isNaN(counter)) {
```

Comment:
It is a very interesting validation. 
![image](https://user-images.githubusercontent.com/80256770/205327504-db412749-fe79-4da7-9f32-e9ddd146a0cc.png)